tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -166,6 +180,10 @@ def _return_class_from_tap(self, return_class='ContactDetails'):
                 return EditContact(self.marionette)
             elif return_class == 'SelectContact':
                 return None
+            elif return_class == 'SetIceContacts':
```

Comment:
I checked the DOM of the system app and contacts, everything is done withing the Contacts app. So there is no switch to do.

I checked by removing this line locally and the test passed.